src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+using CppSharp.Generators.CSharp;
+using System.Text.RegularExpressions;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass,ICommentVisitor<bool>
+    {
+        public bool VisitBlockCommand(BlockCommandComment comment)
+        {
+            return true;
+        }
+
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (!base.VisitDeclaration(decl))
+                return false;
+
+            if (decl.Comment != null)
+            {
+                var fullComment = decl.Comment.FullComment;
+                VisitFull(fullComment);
+                
+            }
+            return true;
+        }
+
+        public bool VisitFull(FullComment comment)
+        {
+            foreach (var block in comment.Blocks)
+                block.Visit(this);
+
+            return true;
+        }
+        #region Comments Visit
+        public bool VisitHTMLEndTag(HTMLEndTagComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitHTMLStartTag(HTMLStartTagComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitInlineCommand(InlineCommandComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitParagraphCommand(ParagraphComment comment)
+        {
+            Regex regex = new Regex(@"^(<|>).{0,}(|_|-)");
+            foreach (var item in comment.Content)
+            {
+                switch (item.Kind)
+                {
+                    case DocumentationCommentKind.TextComment:
+                        TextComment com = (TextComment)item;
+                        if (!regex.IsMatch(com.Text))
+                            break;
+                        else
+                        {
+                            if (com.Text.StartsWith("<", StringComparison.Ordinal))
+                                com.Text = $"{com.Text}{">"}";
+                            else if (com.Text.StartsWith(">", StringComparison.Ordinal))
+                                com.Text = com.Text.Substring(1);
+                            break;
+                        }
+                    case DocumentationCommentKind.FullComment:
```

Comment:
No need to enumerate all types just to write a comment. Adding it to the default section suffices.